pkg/controller/plan/adapter/ocp/builder.go

```diff
@@ -525,6 +478,35 @@ func pvcSourceName(namespace, name string) string {
 	return fmt.Sprintf("%s/%s", namespace, name)
 }
 
+func (r *Builder) SupportsVolumePopulators() bool {
+	return false
+}
+
+func (r *Builder) EnsurePopulatorVolumes(vmRef ref.Ref, annotations map[string]string, secretName string) (pvcNames []string, err error) {
+	err = planbase.VolumePopulatorNotSupportedError
+	return
+}
+
+func (r *Builder) PrePopulateActions(c planbase.Client, vmRef ref.Ref) (ready bool, err error) {
```

Comment:
that's not part of the interface of the builder and looks like it's not being called, is it?

Reply:
Yeah, that method was renamed back to `PreTransferActions`  and it goes in the `Client` interface. I will remove it.